packages/appstore/src/components/loader/trading-app-skeleton/tradingAppSkeleton.scss

```diff
@@ -0,0 +1,58 @@
+@import '../skeletonLoader.scss';
+
+.trading-app-card-loader {
+    display: flex;
+    gap: 24px;
+    padding: 1rem 0rem;
+
+    &__icon {
+        height: 48px;
+        border-radius: $BORDER_RADIUS_2;
+        width: 48px;
+    }
+
+    &__container {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        &__content {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            gap: 4px;
+            &--details {
+                display: flex;
+                flex-direction: column;
+                gap: 4px;
+
+                &__title {
+                    height: 20px;
+                    border-radius: $BORDER_RADIUS_2;
+                    width: 75px;
+                    @include mobile {
+                        width: 88px;
+                        height: 18px;
+                    }
+                }
+
+                &__description {
+                    height: 18px;
+                    border-radius: $BORDER_RADIUS_2;
+                    width: 191px;
+                    @include mobile {
+                        width: 181px;
+                        height: 12px;
```

Comment:
Not sure why the values being hardcoded in px, can you share a video of the responsive experience?

Reply:
@mahdiyeh-deriv these values are from design and there are no skeletons are sharing the same sizes, even two of them.

https://github.com/binary-com/deriv-app/assets/82078941/5c64a0eb-c6ef-45d0-956f-bd2c9dbee8f8